pom.xml

```diff
@@ -688,14 +689,26 @@
         <artifactId>selenium-java</artifactId>
         <version>${selenium.version}</version>
         <scope>test</scope>
+        <exclusions>
+          <exclusion>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-remote-driver</artifactId>
+          </exclusion>
+        </exclusions>
+      </dependency>
+      <dependency>
+        <groupId>org.seleniumhq.selenium</groupId>
+        <artifactId>selenium-remote-driver</artifactId>
```

Comment:
Explicit management `selenium-remote-driver` because it requires exclude netty, guava, bytebuddy as before

Reply:
Why not just only exclude netty, guava, and bytebuddy rather than selenium-remote-driver?

Reply:
When using selenium 3.1, netty, guava, and bytebuddy` is the direct dependency of `selenium-java`, so we can directly exclude them.

But after migrating to selenium 4.2 , `netty, guava, and bytebuddy` is directly dependency of `selenium-remote-driver`, not `selenium-java`.

Reply:
Hmm, I think we can exclude indirect dependencies.
I confirmed with `build/mvn -Phive -Phive-thriftserver -DskipTests dependency:tree`.

`jackson-module-scala` also excludes an indirect dependency.
https://github.com/apache/spark/blob/a2304e45d31d3d802a3976b9cd1ea0f72a2d604b/pom.xml#L1001-L1011

Reply:
Amazing!!!  Let me have a try

Reply:
like 

```
<dependency>
        <groupId>org.seleniumhq.selenium</groupId>
        <artifactId>selenium-java</artifactId>
        <version>${selenium.version}</version>
        <scope>test</scope>
        <exclusions>
          <exclusion>
            <groupId>com.google.guava</groupId>
            <artifactId>guava</artifactId>
          </exclusion>
          <exclusion>
            <groupId>io.netty</groupId>
            <artifactId>*</artifactId>
          </exclusion>
          <exclusion>
            <groupId>net.bytebuddy</groupId>
            <artifactId>byte-buddy</artifactId>
          </exclusion>
        </exclusions>
      </dependency>
```
?

Reply:
Yes. But as you noticed, we may need some additional exclusions.